sdcm/provision/azure/utils.py

```diff
@@ -93,11 +94,49 @@ def get_scylla_images(  # pylint: disable=too-many-branches,too-many-locals
     return output[::-1]
 
 
+def get_released_scylla_images(  # pylint: disable=unused-argument
+        scylla_version: str,
+        region_name: str,
+        arch: VmArch = VmArch.X86,
+        azure_service: AzureService = AzureService()
+
+) -> list[CommunityGalleryImageVersion]:
+    branch_version = '.'.join(scylla_version.split('.')[:2])
+    if is_enterprise(branch_version):
+        gallery_image_name = f'scylla-enterprise-{branch_version}'
+    else:
+        gallery_image_name = f'scylla-{branch_version}'
+    community_gallery_images = azure_service.compute.community_gallery_image_versions.list(
+        location=region_name,
+        gallery_image_name=gallery_image_name,
+        public_gallery_name='scylladb-7e8d8a04-23db-487d-87ec-0e175c0615bb',
```

Comment:
@yaronkaikov 

is there a place we can get this value ? (would it ever gonna change ?)

Reply:
It is not going to change. we have it hard coded for promotion as well